packages/terra-status/tests/jest/Status.test.jsx

```diff
@@ -38,9 +38,9 @@ it('should render a arrange with status and customProps', () => {
 it('should have all props including customProps set correctly', () => {
   const arrangeWithStatus = <Status color="green" id="id" >{arrange}</Status>;
   const wrapper = shallow(arrangeWithStatus);
```

Comment:
I think we can update this to a screenshot & jest will do this comparison for us